packages/host/app/lib/stack-item.ts

```diff
@@ -1,19 +1,21 @@
-import { type Deferred, apiFor } from '@cardstack/runtime-common';
+import {
+  type Deferred,
+  type LooseSingleCardDocument,
+} from '@cardstack/runtime-common';
 
 import {
   type CardResource,
   getCard,
 } from '@cardstack/host/resources/card-resource';
 
 import type { CardDef, Format } from 'https://cardstack.com/base/card-api';
-import type * as CardAPI from 'https://cardstack.com/base/card-api';
 
 interface Args {
   format: Format;
   owner: object;
   request?: Deferred<CardDef | undefined>;
   stackIndex: number;
-  card?: CardDef;
```

Comment:
this is super problematic--we should not allow card instances to be passed into the StackItems since we don't know the provenance of the instance (e.g. did it come from a resource?). Rather the StackItem should be responsible for getting the card instance from the CardResource.